ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -18,10 +18,20 @@ import {
   validateGeo,
```

Comment:
✏️ Might just be preference, but I would create a line folder, put this in it, and factor out all of the individual components so this file is more searchable and readable.